### ISSUE TITLE
[Feature Request] Global default settings

### ISSUE THREAD

**Issue by 38** [38/plotters#73]
We need some default global setting, such as default styles for different things, etc.

**Comment by TatriX** [comment]
Are you thinking about global shared (optionally mutable) state or more about having config per chart context?

**Comment by TatriX** [comment]
I think embedding configuration in context object and implementing default for it should do the trick.

**Comment by 38** [comment]
Yes, that's nice suggestion. But I really think something may beyond that.

The first time I thought about this is because somebody reports on OSX it doesn't work because `FontNotFound` error, I want to have a global default fallback font config. But I don't see any motivation for now, especially we have generic font family support and switched to font-kit, a better maintained font loading crate.

But I am stilling thinking another thing: currently we need to call `DrawingArea::fill` before drawing especially for backend. I think this is a little bit annoying.  
Also I feel annoying for each time I need to disable the mesh line, but at the same time I feel a little bit hesitated to turn it off by default. 

All this things make me think we should have a global table for configurations - But haven't come up with a concrete idea.
